docs/zh/troubleshoot.md

```diff
@@ -1,7 +1,3 @@
-<template>
-  <outdated class="blue"></outdated>
-</template>
-
 # 故障排查
```

Comment:
Could these be removed from other language files so they don't appear twice?

Reply:
Other language files haven't used this template yet, so the notification will appear once.

Reply:
Isn't it being used in the Italian version?

Reply:
No, it isn't. I searched this template code in Italian doc, and found nothing.